apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,321 @@
+        $('#confirmDeleteButton').on('click', function () {
+          
+          var button = $(this);
+          var policyId = button.data('policy-id'); 
+         
+          $('form[name="delete-override-form"] input[name="policy_id"]').val(policyId);
+          // Set the __action value to "delete_override"
+          $('form[name="delete-override-form"] input[name="__action"]').val('delete_override');
+
+          // Submit the form
+          $('form[name="delete-override-form"]').submit();
+          
+        });
+    });
+
+  </script>
+  <%- include('../partials/navbar'); %>
+  <main id="content" class="container-fluid">
+    <div class="card mb-4">
+      <div class="card-header bg-primary text-white d-flex align-items-center justify-content-between">
+        Access Overrides - <%= assessment.title %>
+        <% if (authz_data.has_course_instance_permission_edit) { %>
+        <button type="button" class="btn btn-light" data-toggle="modal" data-target="#addPolicyModal" >
+          Add <i class="fas fa-plus"></i>
+        </button>
+        <% } %>
+      </div>
+      <div class="modal fade" id="addPolicyModal" tabindex="-1" role="dialog" aria-labelledby="addPolicyModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="addPolicyModalLabel">Add New Override</h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              <form id="add-new-override" method="POST">
+                <input type="hidden" name="__action" value="add_new_override">
+                <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                <input type="hidden" class="form-control" id="current_student_uid" name="current_student_uid" value="<%= user.uid %>" required readonly>
+                <% if (!assessment.group_work) { %>
+                  <div class="form-group">
+                    <label for="student_uid">Student UID</label>
+                    <input type="text" class="form-control" id="student_uid" name="student_uid" placeholder="student@example.com" required>
+                  </div>
+                <% } else { %>
+                  <div class="form-group">
+                    <label for="group_id">Group Name</label>
+                    <input type="text" class="form-control" id="group_name" name="group_name" required>
+                  </div>
+                <% } %>
+                <div class="form-group">
+                  <label for="start_date">Start Date</label>
+                  <div class="input-group">
+                    <input type="datetime-local" class="form-control" id="start_date" name="start_date" required>
```

Comment:
You should add `step="1"` to all `<input type="datetime-local" ...>` to allow users to select seconds. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local#step:

> For `datetime-local` inputs, the value of `step` is given in seconds, with a scaling factor of 1000 (since the underlying numeric value is in milliseconds). The default value of `step` is 60, indicating 60 seconds (or 1 minute, or 60,000 milliseconds).

Reply:
done in https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/9b67d9aec841c4a2f01ed2618e8f3fcf13c36708